src/parse/ris_live.go

```diff
@@ -79,44 +80,49 @@ type RisLiveMessage struct {
 }
 
 // connects to ris live, starts go routine receiverHandler, manages connection and subscription
-func ParseRisLiveData(msgChannel chan common.Message, config *config.Configuration) error {
+func ParseRisLiveData(msgChannel chan common.Message, config *config.Configuration) {
 
 	fmt.Println("starting...")
 
+	//for each subscription
+	for _, subscription := range config.Subscriptions {
+		//go routine handleSubscription
+		go handleSubscription(msgChannel, subscription)
+	}
+
+	// alternatives:
```

Comment:
Suggest removing these old comments, but also, it will be good to document a few example JSON filters for RIS Live, plus a pointer to the RIS Live manual somewhere. Two places that would make sense would be in the code near where the subscriptions are parsed, and also the README.